nemo_skills/pipeline/run_cmd.py

```diff
@@ -76,32 +99,80 @@ def run_cmd(
         "--not_exclusive",
         help="If --not_exclusive is used, will NOT use --exclusive flag for slurm",
     ),
+    get_random_port: bool = typer.Option(False, help="If True, will get a random port for the server"),
```

Comment:
Why do we need this parameter? E.g. all other scripts don't have it and I think the default logic of doing

```
get_random_port = server_gpus != 8 and not exclusive
```

covers majority of possible cases? If you really think we should have it, we should then add it to all scripts, not just this one

Reply:
In run cmd, i want it to be fully flexible for dev to control their env. The above logic is fine for our slurm env, may not be the case for others.

Reply:
As to other scripts - it depends. Do we want users to bother with setting this for the most common use cases (like our clusters?) - probably not. So I'm in favor of keeping those automatic. This one is too generic of a script - broader responsibility than just our clusters. So it makes sense to be more flexible in this one.